metadata/demographic/metro_micro_statistical_areas.md

```diff
@@ -0,0 +1,72 @@
+# Title
+
+Utah Metro Micro Statistical Areas
+
+## ID
+
+4e8eaf1f-eda3-47bb-974f-f99be23847b3
+
+## Brief Summary
+
+Polygon dataset of geographic entities created by the US Office of Management and Budget for statistical purposes.
+
+## Summary
+
+This dataset contains polygons representing core based statistical areas as delineated by the [US Office of Management and Budget](Polygon dataset of geographic entities created by the US Office of Management and Budget for statistical purposes.).
+
+## Description
+
+### What is the dataset?
+
+Metropolitan Statistical Areas and Micropolitan Statistical Areas are both types of core based statistical areas (CBSAs) defined by the United States Office of Management and Budget. A CBSA is an area where at least 50% of the population is living in an urban area. Metropolitan Statistical Areas contain at least 50,000 people and Micropolitan Statistical Areas contain at least 10,000 people. This dataset displays both types of CBSA as polygons.
+
+### What is the purpose of the dataset?
+
+These data are used by federal agencies for collecting, tabulating, and publishing federal statistics.
+
+### What does the dataset represent?
+
+Each polygon in this dataset indicates the geographic extent of a CBSA. Features in this dataset include the name, land area, water area, and whether the polygon represents a metro or micro statistical area.
+
+### How was the dataset created?
+
+UGRC derived this layer from the nationwide Topologically Integrated Geographic Encoding and Referencing (TIGER) dataset. You can learn more about TIGER data on the United States Census Bureau [website](https://www.census.gov/geographies/mapping-files/time-series/geo/tiger-line-file.html). This layer has been clipped to show core based statistical areas in Utah only.
+
+### How reliable and accurate is the dataset?
+
+This dataset represents all CBSAs currently defined in Utah. Please reach out to [our team](https://gis.utah.gov/contact/) with questions or concerns about this dataset.
+
+## Credits
+
+### Data Source
+
+US Census Bureau
+
+### Host
+
+UGRC
+
+## Restrictions
+
+## License
+
+## Tags
+
+- US Census Bureau
+- Population
+- Statistics
+- Office of Management and Budget
+
+## Secondary Category
+
+## Data Page Link
+
+## Update
+
+### Update Schedule
+
+Static
```

Comment:
I think we should change this to "As Needed" or the equivalent wording that you've used on other datasets.  It changes periodically and because this dataset doesn't have a date in the name, we'll update when we hear of a new release.